contracts/dao/voting/credits-vault/src/contract.rs

```diff
@@ -95,32 +84,25 @@ pub fn execute_update_config(
     deps: DepsMut,
     info: MessageInfo,
     new_credits_contract_address: Option<String>,
-    new_owner: Option<Admin>,
+    new_owner: String,
```

Comment:
why did you change `Admin` to `String`?

Reply:
long story short: so it's **simpler** and **common**

**simpler:** because it's anyway eventually stored as `Addr` in the contract's cfg. just look a bit below at how this value was handled and how it's now handled. `Admin` struct is very complicated to be passed, just look at the changes here: https://github.com/neutron-org/neutron/pull/166/files. if we pass `Admin::CoreModule`, it simply sets the owner to the sender, so why don't allow sending the simple sender address?

**common:** another reason is that we have the owner field as String in all other contracts

Reply:
okay, i prefer to trust you!